test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -1692,7 +1692,7 @@ void ProtoConverter::fillFunctionCallInput(unsigned _numInParams)
 		case 2:
 			m_output << "sload(" << slot << ")";
 			break;
-		case 3:
+		default:
```

Comment:
Wow! Interesting change. I would have preferred the former, though.

Reply:
You're right. May be I should retain case 3 and add a default with a false assertion? Sometimes clang tidy does not know what it is doing ;-)

Reply:
It is semantically correct, isn't it? I was impressed that clang-tidy could understand that `diceValue` is the set `{0, 1, 2, 3}`. Adding the assert would be a bit weird here, since it's obviously never going to be triggered :)

Reply:
sorry, if I wasn't clear. clang-tidy actually asked me to add a default case over and above case 3 but I decided to simply do a default. I'm pretty sure it does not perform data flow analysis of the `dice` modulo statement in case that is what you were referring to.

Reply:
You are right about not wanting to have a redundant assertion here. Will merge this PR :-)

Reply:
Ah, I see. In that case, maybe a default makes sense :)

Reply:
Can `clang-tidy` make its own replacements?

Reply:
Yeah, but recently I introduced a bug by auto accepting all its suggestions so I don't have a good feeling about it. More specifically, this is what happened

old code:
```
string x = "something";
vector<string> y;
y.emplace_back(x + "string_literal");
```

clang-tidy suggested replacing `x + "string_literal"` with `x.append("string_literal:)` which I accepted without thinking too much about the semantics.

Turns out `append` not only returns the appended string but also modifies the original string which I did not originally intend (and hence the temporary in the original code) :(

clang-tidy felt too many temporaries (as in the string addition and emplace) is bad